avm/res/web/serverfarm/tests/e2e/defaults/main.test.bicep

```diff
@@ -0,0 +1,60 @@
+targetScope = 'subscription'
+
+metadata name = 'Using default parameter set'
+metadata description = 'This instance deploys the module with a base set of parameters. Note it does include the use of Availability zones by default.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-web.serverfarms-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'wsfmin'
+
+@description('Optional. A token to inject into the name of each resource. This value can be automatically injected by the CI.')
+param namePrefix string = '#_namePrefix_#'
+
+// ============ //
+// Dependencies //
+// ============ //
+
+// General resources
+// =================
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+  name: resourceGroupName
+  location: location
+}
+
+// ============== //
+// Test Execution //
+// ============== //
+
+@batchSize(3)
+module testDeployment '../../../main.bicep' = [for iteration in [ 'init', 'idem' ]: {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name, location)}-test-${serviceShort}-${iteration}'
+  params: {
+    name: '${namePrefix}${serviceShort}001'
+    location: location
+    sku: {
+      name: 'P1v3'
+      tier: 'Premium'
+      size: 'P1v3'
+      family: 'P'
+      capacity: 3
+    }
+    zoneRedundant: true
+    kind: 'App'
```

Comment:
```suggestion
```

Defaults to 'App' anyways and should be removed to align with the purpose of the 'defaults' test